specification/dimensions/chargedescription.md

```diff
@@ -0,0 +1,30 @@
+# Charge Description
+
+A Charge Description provides a high-level context of a line item without requiring additional discovery.  This column is typically a brief, human-readable summary of a select group of corresponding details across a cost and usage dataset or provides information not otherwise available.
```

Comment:
This isn't very explicit and will likely result in wildly different implementations making it useless for practitioners. Can we be more specific in what we want to be here and what it's for? "High-level context" could be anything.

I'd like to be explicit about the (at least conceptual) length so we don't end up with paragraphs. I believe the intent is for this to be a one-liner, right? Suggesting "single sentence" since "one-liner" is also a bit ambiguous, but there are at least rules around sentences. I'd be happy with either.
```suggestion
A Charge Description provides a self-contained, summary explanation of what the charge is for and how it was priced in a single sentence. The Charge Description depicts the SKU that was used or purchased, applicable discounts, and any attributes that impact the price relative to other variations of the same SKU (e.g., different regions). The column should be able to stand on its own as a clear description of what is being charged for without any additional columns. The exact price, quantity, and cost do not need to be included.
```

Reply:
I really like this.
Speaking to the difference between ProductName/MeterName vs sku.description etc etc, I think we have a responsibility to set examples for what good SKU descriptions look like.

Not to throw barbs, but AWS is a biggest violator of the standard you have put forth here. A single line item description can cover 3 usage tiers of the exact same thing. Free tier, level 1 usage, and level 2 usage. So I think it is important for us to word it like you have, but also keep in mind that AWS will want to do their tiered pricing and it should be possible with our schema to still do that in a better way than it is being done today in CUR.

Reply:
I like and will incorporate this portion, "... self-contained, summary explanation of what the charge is for and how it was priced ...".  I think the rest is too oriented around cloud-based cost/usage though and don't apply broadly to the SaaS intention.

Reply:
So I think I would add to your paragraph to call it out explicitly - If there are multiple pricing tiers for the same type of service usage, then, that should all fold into one charge description, and XXX column (what would it be?) should be used to distinguish between the pricing tiers.

Reply:
@joshk0 @flanakin 

How about ...

Charge Description provides a self-contained, summary explanation of what the charge is and how it was priced, without requiring additional discovery.  This column is typically a brief, human-readable summary of a select group of corresponding details across a cost and usage dataset or provides information not otherwise available.

1. This keeps the description at a high-level (not just a CSP column)
2. It better defines "high-level" with "self-contained, summary explanation of what the charge is and how it was priced"
3. "how it was priced" implicitly encapsulate Josh's point around multiple tiers without diving into these details.  For v1.0 and our current timeline, I don't want to delineate between cloud, SaaS, and potentially other sources but we can do this for a post-v1.0.